.github/dependabot.yml

```diff
@@ -6,4 +6,20 @@ updates:
   - package-ecosystem: "cargo"
     directory: "/"
     schedule:
-      interval: "weekly"
\ No newline at end of file
+      interval: "weekly"
+
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    target: "dev"
+    ignore:
+      - dependency-name: "@dfinity/*"
+      - dependency-name: "@types/*"
+      - dependency-name: "@typescript-eslint/*"
+      - dependency-name: "eslint"
+      - dependency-name: "eslint-*"
+      - dependency-name: "sass"
```

Comment:
I think we can try to not ignore sass, svelte, and vite. WDYT?

Reply:
very brave eheh! agreed and done